deploy/crd/configauditreports.crd.yaml

```
@@ -20,3 +20,23 @@ spec:
       - all
     shortNames:
       - configaudit
+  additionalPrinterColumns:
+    - JSONPath: .report.scanner.name
+      type: string
+      name: Scanner
```

```suggestion
      name: Scanner
      description: The name of the config audit scanner
```

To match what we have for vuln scanners. Maybe we should add descriptions for the pass/warning/dangerCount fields too

---

Added descriptions for all columns